tests20/python_client/testcases/test_search.py

```diff
@@ -135,559 +134,636 @@ def clear_env(self):
             assert not ("search_collection" in res)
         log.info("clear_env: cleared env (deleted %s search collections)" % count)
 
+    def search_results_check(self, search_res, nq=default_nq, limit=default_limit):
```

Comment:
can we make this method as a check task in class ResponseChecker

Reply:
OK